sequel-enum_values.gemspec

```diff
@@ -45,7 +45,7 @@ Gem::Specification.new do |spec|
 	spec.add_development_dependency 'rspec', '~> 3.9'
 	spec.add_development_dependency 'simplecov', '~> 0.18.0'
 
-	spec.add_development_dependency 'rubocop', '~> 0.88.0'
+	spec.add_development_dependency 'rubocop', '~> 0.89.0'
```

Comment:
Is it required? Seems not related to this PR.

Reply:
i had error with rubocop 0.88.

```
[2020-09-22 22:46:32   ANY]  rubocop
Inspecting 7 files
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/.toys.rb:3:0.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/.toys.rb:6:1.
To see the complete backtrace run rubocop -d.
..An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/lib/sequel/plugins/enum_values.rb:78:23.
To see the complete backtrace run rubocop -d.
...An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:111:6.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:126:6.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:184:6.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:194:6.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:204:6.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:218:6.
To see the complete backtrace run rubocop -d.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:241:5.
To see the complete backtrace run rubocop -d.
..

7 files inspected, no offenses detected

10 errors occurred:
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/.toys.rb:3:0.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/.toys.rb:6:1.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/lib/sequel/plugins/enum_values.rb:78:23.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:111:6.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:126:6.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:184:6.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:194:6.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:204:6.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:218:6.
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/spec/sequel/plugins/enum_values_spec.rb:241:5.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
0.88.0 (using Parser 2.7.1.4, rubocop-ast 0.4.2, running on ruby 2.7.0 x86_64-linux)
```

or with additional information

```
$ toys rubocop
[2020-09-22 22:50:15   ANY]  rubocop -d
For /home/iva/project/sequel-enum_values: configuration from /home/iva/project/sequel-enum_values/.rubocop.yml
configuration from /home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-performance-1.8.1/config/default.yml
configuration from /home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-performance-1.8.1/config/default.yml
Default configuration from /home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/config/default.yml
configuration from /home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-rspec-1.43.2/config/default.yml
configuration from /home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-rspec-1.43.2/config/default.yml
Inspecting 7 files
Scanning /home/iva/project/sequel-enum_values/.toys.rb
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/.toys.rb:3:0.
can't modify frozen Array: [s(:sym, :bundler), s(:hash,
  s(:pair,
    s(:sym, :static),
    s(:true)))]
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/mixin/check_line_breakable.rb:174:in `pop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/mixin/check_line_breakable.rb:174:in `process_args'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/mixin/check_line_breakable.rb:47:in `extract_breakable_node'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/layout/line_length.rb:102:in `check_for_breakable_node'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/layout/line_length.rb:75:in `on_potential_breakable_node'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:91:in `block (2 levels) in trigger_responding_cops'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:113:in `with_cop_error_handling'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:90:in `block in trigger_responding_cops'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:89:in `each'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:89:in `trigger_responding_cops'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:61:in `block (2 levels) in <class:Commissioner>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:59:in `block in on_begin'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:59:in `each'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:59:in `on_begin'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:14:in `walk'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:74:in `investigate'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/team.rb:151:in `investigate_partial'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/team.rb:83:in `investigate'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:295:in `inspect_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:245:in `block in do_inspection_loop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:277:in `block in iterate_until_no_changes'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:270:in `loop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:270:in `iterate_until_no_changes'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:241:in `do_inspection_loop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:121:in `block in file_offenses'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:146:in `file_offense_cache'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:120:in `file_offenses'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:111:in `process_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:90:in `block in each_inspected_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `each'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `reduce'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `each_inspected_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:78:in `inspect_files'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:39:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/command/execute_runner.rb:21:in `execute_runner'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/command/execute_runner.rb:13:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/command.rb:10:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/environment.rb:17:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli.rb:65:in `run_command'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli.rb:72:in `execute_runners'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli.rb:41:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/exe/rubocop:13:in `block in <top (required)>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/2.7.0/benchmark.rb:308:in `realtime'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/exe/rubocop:12:in `<top (required)>'
/home/iva/.rbenv/versions/2.7.0/bin/rubocop:23:in `load'
/home/iva/.rbenv/versions/2.7.0/bin/rubocop:23:in `<main>'
An error occurred while Layout/LineLength cop was inspecting /home/iva/project/sequel-enum_values/.toys.rb:6:1.
can't modify frozen Array: [s(:sym, :exec), s(:hash,
  s(:pair,
    s(:sym, :exit_on_nonzero_status),
    s(:true)),
  s(:pair,
    s(:sym, :log_level),
    s(:const,
      s(:const, nil, :Logger), :UNKNOWN)))]
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/mixin/check_line_breakable.rb:174:in `pop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/mixin/check_line_breakable.rb:174:in `process_args'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/mixin/check_line_breakable.rb:47:in `extract_breakable_node'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/layout/line_length.rb:102:in `check_for_breakable_node'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/layout/line_length.rb:75:in `on_potential_breakable_node'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:91:in `block (2 levels) in trigger_responding_cops'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:113:in `with_cop_error_handling'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:90:in `block in trigger_responding_cops'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:89:in `each'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:89:in `trigger_responding_cops'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:61:in `block (2 levels) in <class:Commissioner>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:150:in `on_if'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:173:in `on_block'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:59:in `block in on_begin'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:59:in `each'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:59:in `on_begin'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-ast-0.4.2/lib/rubocop/ast/traversal.rb:14:in `walk'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:74:in `investigate'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/team.rb:151:in `investigate_partial'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cop/team.rb:83:in `investigate'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:295:in `inspect_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:245:in `block in do_inspection_loop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:277:in `block in iterate_until_no_changes'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:270:in `loop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:270:in `iterate_until_no_changes'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:241:in `do_inspection_loop'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:121:in `block in file_offenses'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:146:in `file_offense_cache'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:120:in `file_offenses'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:111:in `process_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:90:in `block in each_inspected_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `each'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `reduce'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `each_inspected_file'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:78:in `inspect_files'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/runner.rb:39:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/command/execute_runner.rb:21:in `execute_runner'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/command/execute_runner.rb:13:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/command.rb:10:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli/environment.rb:17:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli.rb:65:in `run_command'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli.rb:72:in `execute_runners'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/lib/rubocop/cli.rb:41:in `run'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/exe/rubocop:13:in `block in <top (required)>'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/2.7.0/benchmark.rb:308:in `realtime'
/home/iva/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.88.0/exe/rubocop:12:in `<top (required)>'
/home/iva/.rbenv/versions/2.7.0/bin/rubocop:23:in `load'
/home/iva/.rbenv/versions/2.7.0/bin/rubocop:23:in `<main>'
```
and found this https://github.com/rubocop-hq/rubocop/issues/8714

Reply:
OK, thank you, I got it, but it's better to do non-related things in separate PRs. You can mark this PR as "blocked" (by editing the title) or convert it to a draft while required other PRs not merged.

So, I've updated dependencies in `master` a0a5cd4 and connected Depfu to this project. Please, rebase.